### PR TITLE
[#168,#169,#205] Remove Ubuntu 16.04, Ubuntu 18.04, and MySQL 5.7

### PR DESCRIPTION
**Pull request by SwooshyCueb**
Addresses #168
Addresses #169
Addresses #205

### CODE REVIEW

File: irods_testing_environment/odbc_setup.py [hunk 1]
```diff
@@ -285,70 +209,14 @@ def configure_odbc_driver_centos_7_mysql_8029(csp_container, odbc_driver):
...
-
 def configure_odbc_driver_ubuntu_2004_mysql_8029(csp_container, odbc_driver):
     """Configure ODBC driver for mysql 8.0 on ubuntu 20.04.
 
     Argument:
     csp_container -- docker container on which the iRODS catalog service provider is running
     odbc_driver -- path to local archive file containing the ODBC driver package
     """
-    if not odbc_driver:
+    if not odbc_dderiver:
```
Comment (korydraughn) [inline irods_testing_environment/odbc_setup.py:219]: odbc_dderiver -> odbc_driver

**Review by korydraughn — APPROVED** [review]
Looks good to me.

Let's get a second approval before pounding.

File: irods_testing_environment/odbc_setup.py (hunk 1 above)
Comment (alanking) [reply irods_testing_environment/odbc_setup.py:219]: Yes, let's fix this

**Review by alanking — COMMENTED** [review]
Looks good to me. We should probably push a tag like we did for the development environment which marks the point where this happened. Some older versions of iRODS are only available on these older platforms so it would be good to know which SHA to check out in case we need it.

File: irods_testing_environment/odbc_setup.py (hunk 1 above)
Comment (SwooshyCueb) [reply irods_testing_environment/odbc_setup.py]: Fixed

**Review by alanking — APPROVED** [review]
\# it

**Comment by alanking** [comment]
I created an issue for the tag thing so we can keep this moving: https://github.com/irods/irods_testing_environment/issues/207

**Comment by SwooshyCueb** [comment]
octothorped!

**Pull request merged**